build.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project name="vufindcode" basedir="." default="main">
+  <property name="sh" value="/bin/sh" />
   <property name="tmp" value="/tmp" />
   <property name="package"  value="${phing.project.name}" override="true" />
   <property name="builddir" value="${tmp}/build/${phing.project.name}" override="true" />
   <property name="srcdir"   value="${project.basedir}" override="true" />
+  <property name="downloadsdir" value="${srcdir}/downloads" override="true" />
```

Comment:
You don't need this downloadsdir property. In the main VuFind build.xml, it is used to download Solr and composer, but we don't do those things here, so it is not used. You can just remove this line. (An easy way to check if a property is needed is to search the whole file for the property name (or, if you want to search more narrowly for something short that matches a lot of text, for `${property_name}`)).